website/src/blogs/2018-08-29-openebs-0-7-release-pushes-cstor-storage-engine-to-field-trials.md

```diff
@@ -0,0 +1,186 @@
+---
+title: OpenEBS 0.7 Release pushes cStor Storage Engine to field trials!
+author: Kiran Mova
+author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
+date: 29-08-2018
+tags: Container, Kubernetes, OpenEBS, Storage Solutions
+excerpt: Before I get into some fascinating features of 0.7 release, I would like to take this opportunity to thank all users who have taken our OpenEBS survey and have come forward to share your Kubernetes Stateful Workload and OpenEBS adoption stories.
+---
+
+Before I get into some fascinating features of 0.7 release, I would like to take this opportunity to thank all users who have taken our OpenEBS survey and have come forward to share your Kubernetes Stateful Workload and OpenEBS adoption stories. When I hear statements like ***“OpenEBS just works!”***, it is definitely a sign that the project and the OpenEBS Developer Community is headed in the right direction. It is both heartening and humbling. Thank you!
+![](https://cdn-images-1.medium.com/max/800/08J2dBBRWCnbu82R_)
```

Comment:
alt text

Reply:
```
https://cdn-images-1.medium.com/max/800/08J2dBBRWCnbu82R_
```
is giving error => `Unable to process`


Please use this one ==> `https://cdn-images-1.medium.com/max/800/0*8J2dBBRWCnbu82R_` and provide `alt` value